curl-sys/build.rs

```diff
@@ -30,7 +31,8 @@ fn main() {
     // Next, fall back and try to use pkg-config if its available.
     match pkg_config::find_library("libcurl") {
         Ok(..) => return,
-        Err(..) => {}
+        Err(e) => writeln!(&mut stderr(), "Couldn't find libcurl from \
+                           pkgconfig ({:?}), compiling it from source...", e).unwrap(),
```

Comment:
I used stderr since I thought stdout in the build script is strictly for cargo configuration and printing arbitrary lines to stdout might confuse cargo. So that's not the case?